### PR TITLE
Remove folder

### PR DESCRIPTION
**Pull request by Rashidovicc**

### CODE REVIEW

**Comment by MrTukhtaboev** [comment]
Chatoguu

**Comment by Rashidovicc** [comment]
> Chatoguu

O'ziz aytingizku

**Comment by MrTukhtaboev** [comment]
Nimaga .csproj da ItemGroup Add folder DTo bolib qolgan? Javob bering?

**Comment by FirdavsIbra** [comment]
Please create an issue before pull request!

File: src/ZaminEducation.Service/ZaminEducation.Service.csproj [hunk 1]
```diff
@@ -6,4 +6,8 @@
     <Nullable>disable</Nullable>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Folder Include="DTOs" />
```
Comment (MrTukhtaboev) [inline src/ZaminEducation.Service/ZaminEducation.Service.csproj:10]: Nima uchun bunday qildingiz? Kerak emas bu line

**Review by MrTukhtaboev — APPROVED** [review]
OK, accepted

**Comment by FirdavsIbra** [comment]
Your pull request was successfuly merged! :)

**Pull request merged**